source/D2Game/include/GAME/Clients.h

```diff
@@ -218,7 +218,7 @@ struct D2ClientStrc
 	uint64_t nPingMovingAverage;				//0x4F8
 	uint32_t dwPingsCount;						//0x500
 	int32_t nExpLoss;							//0x504
-	uint32_t dwExpLost;							//0x508
+	uint32_t dwLocale;							//0x508
```

Comment:
This one is based on how it propagates to CLIENTS_AddToGame from the logon packet.